test/e2e-tests.sh

```diff
@@ -60,6 +60,7 @@ go_test_e2e -timeout=10m \
   ./test/scale || failed=1
 
 # Auto TLS E2E tests mutate the cluster and must be ran separately
+kubectl apply -f ./test/config/autotls/certmanager/selfsigned/
```

Comment:
should this be cleaned up after tests to avoid interfere with tests ran afterwards?

Reply:
I added the step to clean it up.